test/model/game_state_test.dart

```diff
@@ -115,5 +117,48 @@ void main() {
     });
   });
 
+  group('Special cards', ()
+  {
+    test('should put Mutiny in next if too many resources', () {
+      const plunderAWreck = PlunderAWreck();
+      const aRivalShip = ARivalShip();
```

Comment:
I don't think the starting deck is particularly relevant to these tests - `makeGameState` actually initialises it to this if you don't pass anything in. Would make the tests a bit shorter / more focused on the part they're trying to test if we didn't pass a deck in at all.

Reply:
Agreed, done.